src/map/modifier.h

```diff
@@ -955,6 +955,7 @@ enum class Mod
     GEOMANCY_MP_NO_DEPLETE = 1037, // Percent chance for Geomancy to cost 0 MP (GEO AF3 Sets)
 
     DIG_BYPASS_FATIGUE = 1074, // Chocobo digging modifier found in "Blue Race Silks". Modifier works as a direct percent.
+    BREATH_DMG_DEALT = 3033, // Breath damage dealt
```

Comment:
Isnt this id... high?
    //
    // SPARE IDs:
    // 53
    // 74 to 79
    // 138 to 143
    // 156 to 159
    // 217 to 223
    // 261 to 280
    //
    // SPARE = 1075, and onward

Reply:
I got it from [1050007342588112958](https://discord.com/channels/392903136336936960/832349765718900827/1050007342588112958)

Reply:
no idea how that works tbh

Reply:
I've believed in lies

Reply:
It's all a dream from Diabolos anyway